app/src/main/java/com/android/sample/ui/createRecipe/AddInstructionStepScreen.kt

```diff
@@ -27,11 +27,18 @@ import com.android.sample.resources.C.Tag.MINLINES_VISIBLE_FOR_INSTRUCTION
 import com.android.sample.resources.C.Tag.SAVE_BUTTON_TAG
 import com.android.sample.resources.C.Tag.SPACE_BETWEEN_ELEMENTS
 import com.android.sample.resources.C.Tag.TIME_CHARACTER_LIMIT
+import com.android.sample.resources.C.TestTag.AddInstructionStepScreen.DELETE_BUTTON
+import com.android.sample.resources.C.TestTag.AddInstructionStepScreen.ICON_DROPDOWN
+import com.android.sample.resources.C.TestTag.AddInstructionStepScreen.INPUT_CONTAINER
+import com.android.sample.resources.C.TestTag.AddInstructionStepScreen.INSTRUCTION_ERROR
+import com.android.sample.resources.C.TestTag.AddInstructionStepScreen.INSTRUCTION_INPUT
+import com.android.sample.resources.C.TestTag.AddInstructionStepScreen.TIME_INPUT
 import com.android.sample.ui.navigation.NavigationActions
 import com.android.sample.ui.navigation.Route
 import com.android.sample.ui.navigation.Screen
 import com.android.sample.ui.theme.Typography
 import com.android.sample.ui.theme.lightCream
+import com.android.sample.ui.utils.PlateSwipeAlertBox
 import com.android.sample.ui.utils.PlateSwipeButton
 import com.android.sample.ui.utils.PlateSwipeScaffold
 
```

Comment:
Great Jib! Using `checkTimeFormat` is a nice touch for readability, and handling deletion with `isDeleting` and `PlateSwipeAlertBox` is a great addition for user experience. Everything here looks smooth! Honestly great job (I really tried to find issues and couldnt find any, 😒 you are not funny)